vue.config.js

```diff
@@ -31,7 +31,6 @@ module.exports = {
       css: { url: false }
     }
   },
-  lintOnSave: false,
```

Comment:
This looks to be an option of the ESLint plugin for Vue CLI, so I think we can remove it. From https://cli.vuejs.org/core-plugins/eslint.html#configuration:

> The following option is under the section of `vue.config.js`. It is respected only when `@vue/cli-plugin-eslint` is installed.